test/kubemark/common.sh

```diff
@@ -23,3 +24,26 @@ export PROJECT
 MASTER_NAME="${INSTANCE_PREFIX}-kubemark-master"
 MASTER_TAG="kubemark-master"
 EVENT_STORE_NAME="${INSTANCE_PREFIX}-event-store"
+
+RETRIES=3
+
+# Runs gcloudn compute command with the given parameters. Up to $RETRIES will be made
+# to execute the command.
+# arguments:
+# $@: all stuff that goes after 'gcloud compute '
+function run-gcloud-compute-with-retries {
+  local attempt=0
+  while true; do
```

Comment:
Consider using something like

```
for retry in $(seq 1 ${RETRIES}); do
```

See examples in cluster/gce/util.sh.  I think that should simplify the logic a bit.